lib/src/parser.dart

```diff
@@ -1,10 +1,13 @@
-import 'dart:io';
+part of './dotenv.dart';
 
-import 'package:meta/meta.dart';
+abstract class Parser {
```

Comment:
I guess this serve as an interface. So it could be `abstract interface class Parser`